storage/src/main/java/com/google/modernstorage/storage/AndroidFileSystem.kt

```diff
@@ -317,14 +316,13 @@ class AndroidFileSystem(private val context: Context) : FileSystem() {
                 return null
             }
 
-            // DocumentsContract.Document.COLUMN_LAST_MODIFIED
-            val lastModifiedTime = cursor.getLong(0)
-            // DocumentsContract.Document.COLUMN_DISPLAY_NAME
-            val displayName = cursor.getString(1)
-            // DocumentsContract.Document.COLUMN_MIME_TYPE
-            val mimeType = cursor.getString(2)
-            // DocumentsContract.Document.COLUMN_SIZE
-            val size = cursor.getLong(3)
+            val displayName = cursor.getString(cursor.getColumnIndexOrThrow(OpenableColumns.DISPLAY_NAME))
+            val size = cursor.getLong(cursor.getColumnIndexOrThrow(OpenableColumns.SIZE))
```

Comment:
I replaced the two `DocumentsContract.Document.*` column names with `OpenableColumns.*` because the class-doc of `OpenableColumns` makes it clear that document providers are only required to implement these two columns.

I can undo this if you want because they do look slightly inconsistent with `DocumentsContract.Document.COLUMN_LAST_MODIFIED` and `DocumentsContract.Document.COLUMN_MIME_TYPE`.